src/Microsoft.TestPlatform.Common/PublicAPI/PublicAPI.Shipped.txt

```diff
@@ -194,8 +194,6 @@ Microsoft.VisualStudio.TestPlatform.ObjectModel.Engine.IProxyExecutionManager.In
 Microsoft.VisualStudio.TestPlatform.ObjectModel.Engine.IProxyExecutionManager.IsInitialized.get -> bool
 Microsoft.VisualStudio.TestPlatform.ObjectModel.Engine.IProxyExecutionManager.StartTestRun(Microsoft.VisualStudio.TestPlatform.ObjectModel.Client.TestRunCriteria testRunCriteria, Microsoft.VisualStudio.TestPlatform.ObjectModel.Client.ITestRunEventsHandler eventHandler) -> int
 Microsoft.VisualStudio.TestPlatform.ObjectModel.Engine.IProxyTestSessionManager
-Microsoft.VisualStudio.TestPlatform.ObjectModel.Engine.IProxyTestSessionManager.StartSession(Microsoft.VisualStudio.TestPlatform.ObjectModel.Client.ITestSessionEventsHandler eventsHandler) -> bool
```

Comment:
Are we okay with removing public APIs? I'd say breaking changes should probably be reserved for "major" version bump.

Reply:
Yes, we should be ok with breaking the API. In this particular case we don't expect anyone other than us to use this interface (i.e. `IProxyTestSessionManager`) as it's not an extension point and it is buried deep in the `vstest.console` code with no current workflow triggering the usage of Test Session. The entry points of the Test Sessions API (found in `VsTestConsoleWrapper.cs` and `TestSession.cs`) are specifically marked as Obsolete because the API is not final. Furthermore we only expect TE in VS to trigger this workflow (and they haven't implement our changes yet) because they are the primary consumers of our translation layer while `dotnet test` doesn't make use of this layer.